astropy/coordinates/representation.py

```diff
@@ -661,7 +913,8 @@ class CartesianRepresentation(BaseRepresentation):
                                 ('y', u.Quantity),
                                 ('z', u.Quantity)])
 
-    def __init__(self, x, y=None, z=None, unit=None, xyz_axis=None, copy=True):
+    def __init__(self, x, y=None, z=None, unit=None, differentials=None,
+                 xyz_axis=None, copy=True):
```

Comment:
I think I'd put `differentials` behind `xyz_axis`, since that really belongs to the coordinates. Also, just for backward compatibility (I worry much less about someone not using `copy=`...)